core/src/avm2/globals/xml.rs

```diff
@@ -105,11 +105,11 @@ pub fn name_to_multiname<'gc>(
         }
     }
 
-    let name = name.coerce_to_string(activation)?;
-    Ok(if &*name == b"*" {
+    let name_string = name.coerce_to_string(activation)?;
+    Ok(if &*name_string == b"*" || *name == Value::Undefined {
```

Comment:
I am afraid this is not correct. name_to_multiname roughly maps to ToXMLName (10.6 in [ECMA-357](https://www.ecma-international.org/publications-and-standards/standards/ecma-357/)), which says throw for `undefined`. (To be fair we don't correctly do that at the moment either)

Per spec we should be handling undefined in `elements`.